binders/repeat.js

```diff
@@ -88,6 +88,11 @@ module.exports = function() {
 
         this.element.parentNode.insertBefore(frag, this.element.nextSibling);
       }
+
+      // Expose the index
+      this.views.forEach(function(view, i) {
+        view.index = i;
+      });
```

Comment:
This is not needed on populate. It correctly does this at https://github.com/dmvt/built-ins/blob/index/binders/repeat.js#L58 the first time. The problem is only after updates are made to the array.